src/components/gitProvider/GitProvider.tsx

```diff
@@ -589,18 +591,24 @@ function GitForm({
                                         name="auth"
                                         value={value}
                                         checked={value === state.auth.value}
-                                    />{' '}
+                                    />
                                     {Lable}
                                 </label>
                             </Tippy>
                         </div>
                     ))}
                 </div>
-                <div className="flex fs-12 left pt-4 mb-20" style={{ color: '#6b778c' }}>
+                <div className="flex fs-12 left pt-4 mb-16" style={{ color: '#6b778c' }}>
                     <Warn className="icon-dim-16 mr-4 " />
                     Once configured, authentication type cannot be switched from HTTPS (user auth/anonymous) to SSH or
                     vice versa.
                 </div>
+                {state.auth.value === AuthenticationType.ANONYMOUS && (
+                    <div className="pt-10 pb-10 pl-16 pr-16 fs-13 br-4 bw-1 bcb-1 lh-20 cn-9 w-100 eb-2 flexbox mb-40">
```

Comment:
we have one common component ÌnfoColourBar`to show this type of information. Please reuse that

Reply:
Added